tests/main/views/test_auth.py

```diff
@@ -14,19 +14,16 @@
 class TestLogin(BaseApplicationTest):
 
     def setup_method(self, method):
-        super(TestLogin, self).setup_method(method)
-
-        data_api_client_config = {'authenticate_user.return_value': self.user(
-            123, "email@email.com", 1234, 'name', 'name'
-        )}
-
-        self._data_api_client = mock.patch(
-            'app.main.views.auth.data_api_client', **data_api_client_config
```

Comment:
So we do not need `**data_api_client_config` anymore? Just curious :).

Reply:
Nope - I explicitly set the `authenticate_user.return_value` below rather than passing it as args to the patch. There's no real difference between the two, just being consistent with styling in the other apps.

Reply:
cool, thanks for the explanation :). I guess also better to have it explicit, more readable :3.